test/unit/spec/v1/EnrollCall_spec.js

```diff
@@ -598,10 +595,16 @@ Expect.describe('EnrollCall', function() {
   });
 
   describe('Enroll phone number', function() {
+    beforeEach(() => {
+      jest.setTimeout(20000);
+    });
     testEnrollPhoneNumber(setup, resEnrollSuccess);
   });
 
   describe('Verify phone number', function() {
+    beforeEach(() => {
+      jest.setTimeout(20000);
+    });
```

Comment:
Checking also to see if these extra timeouts can be removed

Reply:
Yep these can be removed, pushing update and did a search for all usage of this settimeout to test the removal of those as well 🙏